README.md

```diff
@@ -15,7 +15,7 @@ so you can deploy real infrastructure based on your changes.
 Place this in a `.github/workflows` workflow on your repository:
 
 ```yaml
-- uses: systeminit/actions/trigger@main
+- uses: systeminit/actions@main
```

Comment:
Maybe trigger isn't the right word, but surely we should use some word indicating *what* action you're taking. Most actions I see out there are self documenting by looking at the name; seeing `systeminit/actions` in a workflow tells me nothing about what that workflow does. Might be I've missed your true purpose here though :)

Reply:
If a repo is only going to have a single GitHub Action then it's pointless putting the action.yml inside a subfolder to give the action a name. The normal pattern of a GitHub Action is to not bury it otherwise we would put out src and dist inside the subfolder too

There's 1 entry point for an action to System Initiative :) This gives us that entry point - the name is just a name - we can rename the repo to be actions-trigger if needed but it's kinda a pointless thing to do too